test/Interop/Cxx/extern-var/extern-var.swift

```diff
@@ -23,26 +23,23 @@ ExternVarTestSuite.test("write-from-cxx") {
   expectEqual(84, getCounterFromCxx())
 }
 
-//FIXME mangle non-top-level var names to prevent name collisions
-// ExternVarTestSuite.test("namespaced-write-from-swift") {
-//   Namespaced.counter = 42
-//   expectEqual(42, Namespaced.counter)
-//   expectEqual(42, amespaced.getCounterFromCxx())
-// }
-
-//FIXME mangle non-top-level var names to prevent name collisions
-// ExternVarTestSuite.test("namespaced-write-from-cxx") {
-//   Namespaced.setCounterFromCxx(84)
-//   expectEqual(84, Namespaced.counter)
-//   expectEqual(84, Namespaced.getCounterFromCxx())
-// }
-
-//FIXME mangle non-top-level var names to prevent name collisions
-// ExternVarTestSuite.test("no-collisions") {
-//   counter = 12
-//   Namespaced.counter = 42
-//   expectEqual(12, counter)
-//   expectEqual(42, Namespaced.counter)
-// }
+ExternVarTestSuite.test("namespaced-write-from-swift") {
+  Namespaced.counter = 42
+  expectEqual(42, Namespaced.counter)
+  expectEqual(42, Namespaced.getCounterFromCxx())
+}
+
+ExternVarTestSuite.test("namespaced-write-from-cxx") {
+  Namespaced.setCounterFromCxx(84)
+  expectEqual(84, Namespaced.counter)
+  expectEqual(84, Namespaced.getCounterFromCxx())
+}
+
+ExternVarTestSuite.test("no-collisions") {
+  counter = 12
```

Comment:
Could you add a comment to this test? Something like:

```
// Check that variables with identical names in different namespaces don't collide in any intermediate representation of the compiler.
```

Reply:
Done.